SubterfugeCore/Core/Generation/MapGenerator.cs

```diff
@@ -92,7 +109,9 @@ private List<Outpost> TranslateOutposts(List<Outpost> outposts, RftVector transl
                 newPosition += translation;
 
                 // Add a new outpost to the translated outposts list
-                translatedOutposts.Add(new Outpost(newPosition, outpost.GetOwner(), outpost.GetOutpostType()));
+                Outpost newOutpost = new Outpost(newPosition, outpost.GetOwner(), outpost.GetOutpostType());
```

Comment:
I understand that the question I'll ask here is not directly related to the changes in this PR, but: doesn't this necessarily make an outpost of the same kind as the original outpost that was used as a reference to get to the location of this new outpost? Am I correct in thinking that during generation, outposts are generated kind per kind?

I imagine this might become a problem if we are generating multiple kinds of outposts that should be abundant (factories and generators)?

Reply:
I'm not sure what the question here is? Currently generation does this:
- Generate a single player's outposts (including outpost types)
- Tile and rotate that player's set of outposts into X other starting spots for other players.

I think map generation will be changed in the future.